lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -351,6 +351,12 @@ def add_runtask_options(parser):
                         help="set additional variables as key=value or YAML/JSON, if filename prepend with @", default=[])
 
 
+def add_tasknoplay_options(parser):
+    """Add options for commands that run a task w/o a defined play"""
+    parser.add_argument('--task-timeout', dest="task_timeout", action="store", default=None,
```

Comment:
We say the default is `0` in several spots, yet here we use `None`. It works, just seems odd to not make it `0` for consistency sake.